src/components/view-more/view-more.js

```diff
@@ -0,0 +1,62 @@
+function viewMore() {
```

Comment:
I wonder if we should look at `https://github.com/metmuseum/sculptured/blob/main/sculptured/frontend/components/read-more-overflow/readMoreOverflow.js` ? it has similar logic to this

Reply:
This looks pretty good, although it's hard to totally follow what is happening without seeing it in action. The resize thing is interesting. The bigger thing for me is how can we make it so more of these components are built in Marble natively? Surely this is a perfect candidate. We can talk more about this later.